Docs/Generali/piano_di_progetto/03_pianificazione.tex

```diff
@@ -43,4 +43,19 @@ \section{Pianificazione}
             \caption{Diagramma di Gantt sprint 2}
             \label{fig:3.2} % Etichetta per il riferimento
         \end{figure}
+
+
+        \subsubsection{Sprint 3 (dal 9/12/2024 al 20/12/2024)}
+        In questo terzo sprint continueremo l'aggiornamento delle \textit{"Norme di Progetto"} modificando e migliorando le regole che il team si pone di rispettare per garantire
+        una gestione ottimale delle risorse e un metodo di sviluppo efficace ed efficiente. Successivamente è prevista la stesura delle metriche applicate relative alla qualità del progetto e alla qualità del prodotto all'interno del documento \textit{"Piano di Qualifica"}.
+        Con priorità minore inoltre è prevista la stesura delle metriche di testing sempre all'interno dello stesso dcumento.
```

Comment:
Successivamente, in quanto di minore importanza, è prevista....documento (attento all'errore di battitura)